main.go

```diff
@@ -43,6 +43,19 @@ func init() {
 	flag.StringVar(&localMasterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 }
 
+type leaderConfig struct {
+	LeaseDuration int64
```

Comment:
you can also annotate with `default:"5"` 

and the module will handle it, but I'm fin with it now, we need to release.

Reply:
Actually, I tried it and it did not seem to work when I tested it. So, I modified it this way.

Reply:
oh, ok, makes sense then. I guess the lib is buggy